packages/cascara/src/ui/Columns/Columns.module.scss

```diff
@@ -0,0 +1,15 @@
+$masonry-gutter: 2em;
+$widget-border-radius: 0.5em;
+$widget-background: #f0f0f0;
+
+.Columns {
```

Comment:
@brianespinosa Let me know if this is what we are looking for, here's a screenshot from a standalone window in Cosmos for this change:
![image](https://user-images.githubusercontent.com/13924868/141350084-6febe01d-49ed-428c-b5b5-131b17510449.png)

Reply:
Yes. The column sizing looks great. I am marking this as resolved so it is out of the way when you take a look at the rest of the comments. 👍🏽